### PR TITLE
Scale function app operator search

### PR DESCRIPTION
**Pull request by uolter**

### CODE REVIEW

File: code/env/prod/terraform.tfvars [hunk 1]
```diff
@@ -45,13 +45,17 @@ io_apim_productid         = "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b
...
 redis_cache_sku_name      = "Standard"
+operator_search_capacity  = 10
 operator_search_external_allowed_subnets = [
   # io-backend
   "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-common/providers/Microsoft.Network/virtualNetworks/io-p-vnet-common/subnets/appbackendl1",
   "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-common/providers/Microsoft.Network/virtualNetworks/io-p-vnet-common/subnets/appbackendl2",
   # io apim
   "/subscriptions/ec285037-c673-4f58-b594-d7c480da4e8b/resourceGroups/io-p-rg-common/providers/Microsoft.Network/virtualNetworks/io-p-vnet-common/subnets/apimapi",
 ]
+opertor_search_elastic_instance_minimum      = 10
```
Comment (AleDore) [inline code/env/prod/terraform.tfvars]: typo

File: code/variables.tf [hunk 2]
```diff
@@ -488,8 +488,27 @@ variable "pe_min_csv_rows" {
...
+  default     = 1
+}
+
 variable "operator_search_external_allowed_subnets" {
   type        = list(string)
   description = "External subnets allowed to call operator search function"
   default     = []
 }
+
+variable "opertor_search_elastic_instance_minimum" {
```
Comment (michaeldisaro) [inline code/variables.tf]: typo

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

File: code/env/prod/terraform.tfvars (hunk 1 above)
Comment (uolter) [reply code/env/prod/terraform.tfvars]: thanks .... I'll fix it.

File: code/variables.tf (hunk 2 above)
Comment (uolter) [reply code/variables.tf]: thanks I'll fix it ...

**Pull request merged**